neptune/experiments.py

```diff
@@ -294,13 +311,15 @@ def parameters(self):
 
             Get experiment parameters.
 
-            >>> experiment.parameters
+            >>> experiment.get_parameters()
 
         """
-        return self._simple_dict_to_dataframe(self._leaderboard_entry.parameters)
+        experiment = self._client.get_experiment(self.internal_id)
+        return dict(
```

Comment:
Should we return dict or DataFrame here?

Reply:
Fixed